op-service/eth/ssz.go

```diff
@@ -9,16 +9,33 @@ import (
 	"sync"
 )
 
+type BlockVersion int
+
+const ( // iota is reset to 0
+	BlockV1 BlockVersion = iota
+	BlockV2              = iota
+)
+
 // ExecutionPayload is the only SSZ type we have to marshal/unmarshal,
 // so instead of importing a SSZ lib we implement the bare minimum.
 // This is more efficient than RLP, and matches the L1 consensus-layer encoding of ExecutionPayload.
 
 // All fields (4s are offsets to dynamic data)
-const executionPayloadFixedPart = 32 + 20 + 32 + 32 + 256 + 32 + 8 + 8 + 8 + 8 + 4 + 32 + 32 + 4
+const blockV1FixedPart = 32 + 20 + 32 + 32 + 256 + 32 + 8 + 8 + 8 + 8 + 4 + 32 + 32 + 4
+
+// V1 + Withdrawals offset
+const blockV2FixedPart = blockV1FixedPart + 4
+
+const withdrawalSize = 8 + 8 + 32 + 8
```

Comment:
Could it be `8 + 8 + 20 + 8`?
The withdrawal spec appears to point to the 3rd field being an address. https://github.com/ethereum/consensus-specs/blob/ef434e87165e9a4c82a99f54ffd4974ae113f732/specs/capella/beacon-chain.md#withdrawal

cc @ajsutton

Reply:
Yeah it winds up being `UInt64, Uint64, Vector[byte, 20], UInt64`.  Taku's schema calculates it as having 44 bytes so `8 + 8 + 20 + 8`.

Reply:
Excellent catch -- thank you 🙏 Just fixed this.